index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css"
+      integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY="
+      crossorigin="anonymous"
+    />
+    <title>NFSAS</title>
+  </head>
+  <body>
+    <header>
+      <div id="secondary-nav-bar" class="container">
+        <div class="wrapper">
+          <a href="#"><i class="fa fa-mastodon"></i></a>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </div>
+      </div>
+      <div id="nav-bar">
+        <a href="#" id="logo">NFSAS<span>2022</span></a>
+        <nav id="menu">
+          <span id="menu-close-btn">×</span>
+          <a href="#" class="home-link">Home</a>
+          <a href="#program">Program</a>
+          <a href="#speakers">Speakers</a>
+          <a href="#sponsors">Sponsors</a>
+          <a href="./about.html">About</a>
+        </nav>
+        <a href="#" id="menu-toggler">
+          <svg
+            class="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path stroke-width="3" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </a>
+      </div>
+    </header>
+
+    <section id="section1" class="container">
+      <div class="wrapper">
+        <img src="./assets/img/shape-set-01.png" class="shape-set" alt="" />
```

Comment:
- [ ] [OPTIONAL] Even if your images are visible when I run your project locally, you may experience broken images when you deploy your project, and **this is because not paying attention to the case-sensitivity in your image paths** while referencing an image, you should be explicitly on the image path so that anytime anywhere, your images are visible in the website. To be specific, I would suggest you adjust your image path for example like the above one
From: `./assets/img/shape-set-01.png`  To: `./assets/Img/shape-set-01.png` and this applies to all your images. I hope you get the difference 😉 
![image](https://user-images.githubusercontent.com/90222110/192088687-ca01c27e-d5e7-46cd-b442-252a2409c896.png)

- [ ] [OPTIONAL] Also, please try to add `alt` texts whenever it is necessary for accessibility purposes 😉

Reply:
Thank you I have made all the corrections goin for my final review